### PR TITLE
Docs: adds info on location for mute timings

### PR DESCRIPTION
**Pull request by brendamuir**
support for locations was missing in the docs - added it in to address https://github.com/grafana/support-escalations/issues/8313

### CODE REVIEW

File: docs/sources/alerting/manage-notifications/mute-timings.md [hunk 1]
```diff
@@ -56,18 +56,19 @@ The following table highlights the key differences between mute timings and sile
 
 ## Time intervals
 
-A time interval is a definition for a moment in time. If an alert fires during this interval it will be suppressed. All fields are lists, and at least one list element must be satisfied to match the field. Fields also support ranges using `:` (ex: `monday:thursday`). The fields available for a time interval are: mute timing can contain multiple time intervals. A time interval is a specific duration when alerts are suppressed from firing. The duration typically consists of a specific time range along with days of a week, month, or year.
-
-    All properties for the time interval are lists, and at least one list element must be satisfied to match the field. The fields support ranges using `:` (ex: `monday:thursday`). If you leave a field blank, it will match with any moment of time.
+A time interval is a specific duration when alerts are suppressed from firing. The duration typically consists of a specific time range along with days of a week, month, or year.
```
Comment (chri2547) [inline docs/sources/alerting/manage-notifications/mute-timings.md:59]:
```suggestion
A time interval is a specific duration during which alerts are suppressed. The duration typically consists of a specific time range and the days of the week, month, or year.
```

File: docs/sources/alerting/manage-notifications/mute-timings.md [hunk 2]
```diff
@@ -56,18 +56,19 @@ The following table highlights the key differences between mute timings and sile
...
 Supported time interval options are:
 
 - Time range: The time inclusive of the start and exclusive of the end time (in UTC if no location has been selected, otherwise local time).
+- Location: Depending on the location you select, the time range is displayed in local time.
 - Days of the week: The day or range of days of the week. Example: `monday:thursday`.
 - Days of the month: The date 1-31 of a month. Negative values can also be used to represent days that begin at the end of the month. For example: `-1` for the last day of the month.
 - Months: The months of the year in either numerical or the full calendar month. For example: `1, may:august`.
 - Years: The year or years for the interval. For example: `2021:2024`.
 
+All fields are lists, and at least one list element must be satisfied to match the field. Fields also support ranges using `:` (ex: `monday:thursday`).
```
Comment (chri2547) [inline docs/sources/alerting/manage-notifications/mute-timings.md:70]:
```suggestion
All fields are lists; to match the field, at least one list element must be satisfied. Fields also support ranges using `:` (e.g., `monday:thursday`).
```

**Review by chri2547 — CHANGES_REQUESTED** [review]
a few suggestions

**Review by gillesdemey — APPROVED** [review]
LGTM from my side!

**Comment by brendamuir** [comment]
Also adds a note in response to: https://github.com/grafana/support-escalations/issues/7418

**Review by gillesdemey — APPROVED** [review]

**Review by chri2547 — APPROVED** [review]

**Pull request merged**